configure

```diff
@@ -596,10 +595,10 @@ if [ -z "$CFLAGS" ]; then
    elif [ "$MACHINE" = "mips64" ]; then
       CFLAGS="-O2 -funroll-loops -g $POPCNT_FLAG $ABI_FLAG"
       ANSI_FLAG=""
-   elif [ "$KERNEL" = FreeBSD ]; then
-      CFLAGS="-std=c99 -pedantic -Wall -O2 -funroll-loops -g $POPCNT_FLAG $ABI_FLAG"
+   elif test "$KERNEL" = "FreeBSD" -o "$OS" = "OpenBSD"; then
+      CFLAGS="-std=c99 -pedantic -Wall -O2 -funroll-loops -g $POPCNT_FLAG $ABI_FLAG -I/usr/local/include"
```

Comment:
Why do you need `-I/usr/local/include`? It's not good to hardcode values like this because it hinders cross compiling.

Reply:
Compilers on *BSD don't always look into `/usr/local/include` - and that's where includes for most of the packages are. It's a side effect of them used to build everything using customised makefiles :-(

Also note that these lines are not executed if `CFLAGS` are not `''`, so this is just a sanity fix.
(otherwise one must run ./configure with CFLAGS unset)

Reply:
What do you mean by most packages? Packages like gmp, mpfr? Packages like C standard library?

Reply:
everything what they call "ports" is in /usr/local, e.g. gmp, mpfr (but indeed, not libc).
e.g. on my OpenBSD box:
```
sagemath$ du -h /usr/include | tail -1
46.2M	/usr/include
sagemath$ du -h /usr/local/include | tail -1
183M	/usr/local/include
```

Reply:
Then, I don't know why you need to add `-I/usr/local/include` here. Flint already adds `GMP_INC_DIR` which is going to be `/usr/local/include` by default.

Reply:
ok, right, you convinced me that this should be fixed. Sorry fot confusion.

Reply:
see #756